_posts/2021-07-05-newsletter.md

```diff
@@ -0,0 +1,199 @@
+
+
+## Other changes
+
+Of course, these were not the only [major changes in 0.5](https://github.com/neovim/neovim/commit/a5ac2f45ff84a688a09479f357a9909d5b914294). Here is a short summary of representative new features.
+
+### Decorations
+
+There is an improved "decoration provider" API that allows setting and interacting with "extmarks" (invisible anchored text markers that move when surrounding text is edited), "virtual text" (text overlays that can now be drawn at any position on screen), and highlights. (This API is heavily leveraged by tree-sitter.)
+
+### Floats
+
+The API for floating windows now includes a "z-index" (allowing for control over how floating windows stack) and support for borders.
+
+### Highlight on yank
+
+Neovim now has a built-in function to briefly highlight the yanked region (similarly to https://github.com/machakann/vim-highlightedyank), configurable from Lua. To use it, you can add the following to your `init.vim`:
+```
+au TextYankPost * lua vim.highlight.on_yank {higroup="IncSearch", timeout=150, on_visual=true}
+```
+See `:h vim.highlight.on_yank()` for more configuration options.
+
+### Vim patches
+
+Of the over 4000 commits in this release, about 1000 were patches and runtime updates ported from Vim -- nearly all of them by or with the help of the amazing [@janlazo](https://github.com/janlazo). In particular, the runtime files (syntax files, documentation, etc.) are fully synced with Vim up to [May 2021](https://github.com/neovim/neovim/pull/14424), with many later changes already included as well.
+
+
+## Community
+
+In keeping with the title of this newsletter, one of the most noticeable positive changes was the growth of the community and of new ways of interacting with it.
+
+### Discourse
+
+Previously, support requests and discussions were spread across Reddit, Gitter, and GitHub Discussions and were either ephemeral or hard to search for. We have now consolidated around a new [Neovim Discourse](https://neovim.discourse.group), which is a free and open source forum platform with mailing list and RSS features, in addition to a nice web interface. The Neovim Discourse is an official core project and moderated by core team members.
+
+### Matrix
+
+The official chatroom for Neovim is on [Gitter](https://gitter.im/neovim/neovim). After the acquisition of Gitter by Matrix (a federated chat protocol), this room can now also be accessed from [Matrix](https://app.element.io/#/room/#neovim:matrix.org); it is also bridged to the IRC network [Libera.chat](https://libera.chat). Due to the increasing number of users, there are now additional, more specific rooms for [development of and around neovim](https://app.element.io/#/room/#nvim-dev:matrix.org), [GUIs](https://app.element.io/#/room/#neovim-guis:matrix.org), and [off-topic chat](https://app.element.io/#/room/#neovim-chat:matrix.org) -- with possibly more to come.
```

Comment:
`-- with possibly more to come` is unnecessary.

Reply:
We've talked about having more rooms? But sure, I can remove it.